plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -886,18 +913,32 @@ function MosaicMenu:_recalculateDimen()
 
     -- Create or replace corner_mark if needed
     -- 1/12 (larger) or 1/16 (smaller) of cover looks allright
-    local mark_size = math.floor(math.min(self.item_width, self.item_height) / 16)
+    local mark_size = math.floor(math.min(self.item_width, self.item_height) / 8)
```

Comment:
(May be if over users find the corner mark overwhelming, we could go with 12 as I seem to have found it alright :)

Reply:
I tried 12 but found it too small, but we can always reduce if the people rebel. Otherwise we may have to chip in for an early eye appointment for you and fix the problem from that front 🧐